submitted_models/costar_shafter_sensor_config_1/model.sdf

```diff
@@ -179,25 +179,25 @@
                 <update_rate>30</update_rate>
                 <camera name="realsense_front">
                     <horizontal_fov>1.18682</horizontal_fov>
-                    <lens>
+                    <!--lens>
                         <intrinsics>
                             <fx>617.61480</fx>
                             <fy>617.61480</fy>
                             <cx>321.1996</cx>
                             <cy>234.8967</cy>
                             <s>0</s>
                         </intrinsics>
-                    </lens>
-                    <distortion>
+                    </lens-->
+                    <!--distortion>
                         <k1>0.0</k1>
                         <k2>0.0</k2>
                         <k3>0.0</k3>
                         <p1>0.0</p1>
                         <p2>0.0</p2>
-                    </distortion>
+                    </distortion-->
```

Comment:
Is there a particular reason why `lens` and `distortion` are commented out? I found that the camera points down into the ground with these attributes commented out, which seems incorrect (red is lidar, and white is the depth camera):
![downward_facing_cam](https://user-images.githubusercontent.com/42042756/103939150-577fbc80-50f9-11eb-8c82-4284665af058.png)

If I un-comment `lens` and `distortion`, the camera points slightly upward (see picture below). This seems more correct than down into the ground, but I'm assuming the depth camera should be horizontally aligned (i.e., no pitch), right?
![forward_facing_cam](https://user-images.githubusercontent.com/42042756/103939318-957ce080-50f9-11eb-9200-445c84da1c20.png)